R/predict.R

```diff
@@ -72,6 +72,26 @@ predict.WrappedModel = function(object, task, newdata, subset = NULL, ...) {
     newdata = newdata[subset, , drop = FALSE]
   }
 
+
+  # set factor levels, present in test but missing in train, to NA
+  if (model$learner$fix.factors.prediction == TRUE &&
+      any(vlapply(newdata, function(x) is.factor(x))) &&
+      any(class(model$learner.model) == "lm")) {
```

Comment:
This code should be part of `predictLearner.regr.lm`

Reply:
moved from `predict.WrappedModel()` to `predictLearner.regr.lm`  15b14a4